services/frontend-v3/src/pages/admin/Competency.jsx

```diff
@@ -2,20 +2,16 @@ import React from "react";
 import AdminLayout from "../../components/layouts/adminLayout/AdminLayout";
 import { injectIntl } from "react-intl";
 
-class AdminCompetency extends React.Component {
-  goto = link => this.props.history.push(link);
-
-  render() {
-    return (
-      <AdminLayout
-        changeLanguage={this.props.changeLanguage}
-        displaySideBar={true}
-        type="competency"
-      >
-        {" "}
-      </AdminLayout>
-    );
-  }
+function AdminCompetency(props) {
+  return (
+    <AdminLayout
+      changeLanguage={props.changeLanguage}
+      displaySideBar={true}
+      type="competency"
+    >
+      {" "}
+    </AdminLayout>
+  );
 }
 
 export default injectIntl(AdminCompetency);
```

Comment:
you dont need the injectIntl i dont think. Please remove wereever possible

Reply:
Fixed.